crates/bevy_ui/src/ui_node.rs

```diff
@@ -298,6 +298,75 @@ impl ComputedNode {
 
         clip_rect
     }
+
+    const fn compute_thumb(
+        gutter_min: f32,
+        content_length: f32,
+        gutter_length: f32,
+        scroll_position: f32,
+    ) -> [f32; 2] {
+        if content_length <= gutter_length {
+            return [gutter_min, gutter_min + gutter_length];
+        }
+        let thumb_len = gutter_length * gutter_length / content_length;
+        let thumb_min = gutter_min
+            + scroll_position / (content_length - gutter_length) * (gutter_length - thumb_len);
```

Comment:
Does it make sense to simplify this to: 
```suggestion
            + scroll_position * gutter_length / content_length;
```
? I simplified the terms on the right hand side after `scroll_position` because I thought it looked verbose… and it turns out it was if I am reading it correctly
<img width="355" height="226" alt="Screenshot 2025-11-28 at 6 25 32 PM" src="https://github.com/user-attachments/assets/4460218c-aafb-4ae1-9d00-7e8452ab0063" />
And this makes sense since `gutter_min` should be increased by some